jenkins.sh

```diff
@@ -10,4 +10,5 @@ pip2.7 install sphinx
 make develop extras=[aws,mesos,azure,encryption,cwl]
 export LIBPROCESS_IP=127.0.0.1
 export PYTEST_ADDOPTS="--junitxml=test-report.xml"
+mkdir -p /mnt/ephemeral/tmp && export TMPDIR=/mnt/ephemeral/temp
```

Comment:
Haha we're assuming that there's a mountpoint called /mnt/ephemeral. If there isn't, the directory will still be created but it'll still be on the root volume.